scripts/release.sh

```diff
@@ -48,7 +48,10 @@ for file in "gradle.properties" \
 "example-projects/spock/gradle.properties" \
 "example-projects/testng/gradle.properties" \
 "example-projects/android/gradle.properties" \
+"example-projects/kotlin/gradle.properties" \
 "example-projects/selenium/gradle.properties" \
+"example-projects/spring-boot/gradle.properties" \
+"jgiven-gradle-plugin/gradle.properties"
```

Comment:
Verify that the version is not updated with setting the variable in $root/gradle.properties.